packages/cli/src/scaffold/schema.ts

```diff
@@ -50,21 +54,37 @@ export const generateEventFields = ({
         }),
       ];
 
-export const generateEventType = (event: any, protocolName: string) => `type ${
-  event._alias
-} @entity(immutable: true) {
-      id: Bytes!
-      ${event.inputs
-        .reduce(
-          (acc: any[], input: any, index: number) =>
-            acc.concat(generateEventFields({ input, index, protocolName })),
-          [],
-        )
-        .join('\n')}
-      blockNumber: BigInt!
-      blockTimestamp: BigInt!
-      transactionHash: Bytes!
-    }`;
+      
+const renameInput = (name: string, subgraphName: string) => {
+  const inputMap: BlacklistDictionary = {
+    id: `${subgraphName}_id`
+  } 
+
+  return inputMap[name] ?? name;
+}
```

Comment:
for anyone upgrading we would need to note this is potentially breaking change

Reply:
Yes it is. Is there any convention or anything you use for this types of changes?